### ISSUE TITLE
any plan to release the code of SimpleRE？

### ISSUE THREAD

**Issue by nttmac** [XueFuzhao/GDPNet#2]
Hello, author! 
I read another related paper, "An Embarrassingly Simple Model for Dialogue Relation Extraction". Do you have any plan to release the code?."

**Comment by XueFuzhao** [comment]
Yes, absolutely. We are cleaning the code and try to extend the experiments. I'm also working on a few other deadlines. So there is no clear timeline about that. Actually, the SimpleRE is very very simple and easy to implement. I believe it would not spend too much time to reimplement it on your side. Certainly, if you have no urgent deadline to use the SimpleRE model, you can wait for our code. I promise we will release it ASAP.

**Comment by nttmac** [comment]
> Yes, absolutely. We are cleaning the code and try to extend the experiments. I'm also working on a few other deadlines. So there is no clear timeline about that. Actually, the SimpleRE is very very simple and easy to implement. I believe it would not spend too much time to reimplement it on your side. Certainly, if you have no urgent deadline to use the SimpleRE model, you can wait for our code. I promise we will release it ASAP.

Actually, I have implemented the SimpleRE without RRG. The average and standard deviation of F1 scores are 63.32 and 2.10 respectively, with a gap between the results in your paper 65.5 (0.7).

**Comment by XueFuzhao** [comment]
Thank you for your interest! Please follow the hyperparameters stated in the paper, and add some dropout before classification.  Since DialogRE is not very large, the std would be high if we do not add dropout. BTW, training more epochs may be helpful to get a more stable result.

**Comment by nttmac** [comment]
> Thank you for your interest! Please follow the hyperparameters stated in the paper, and add some dropout before classification. Since DialogRE is not very large, the std would be high if we do not add dropout. BTW, training more epochs may be helpful to get a more stable result.

Yeah! This time, I set the batch size 6, and the results are consistent with those in your paper! thank u for your help and your patience!

**Comment by XueFuzhao** [comment]
So happy to hear that!!! And Yes, we have to set the batch size as 6 in SimpleRE because we have so many labels in each dialogue. Welcome further discussion and collaboration if you are interested in this topic in the future!
